graph/model/user.go

```diff
@@ -0,0 +1,10 @@
+package model
+
+import "gorm.io/gorm"
+
+type User struct {
+	gorm.Model
+	Username string `json:"username" validate:"required"`
```

Comment:
Is it supposed to be `binding:"required"`? That's what the example docs for gin use but I may be tripping. Be good if someone could verify this. I think it'll only change what type of error is returned but it's good to return the proper 400 if there's missing info from the user rather than a 500.

Reply:
That was for the graphQL validation. I'll just move the models to a different folder and remove the graphQL for now as suggested in your comment above. I agree that we shouldn't focus on it for now but I initially added it to keep the feature set from the previous code base.